pandas/core/window/rolling.py

```diff
@@ -1437,7 +1437,11 @@ def count(self):
                 FutureWarning,
             )
             self.min_periods = 0
-        return super().count()
+            result = super().count()
```

Comment:
grr this is not pretty. maybe have a _count() method where you can pass min_periods? (for 1.2.2 this is prob ok, but for 1.3)

Reply:
Hmm either is a shim that will be removed in 2.0. I think in either way I'll have to do this gymnastics since `min_periods` is set globally on the object

Reply:
ok for here, I was talking about a `_count()` (private method) that we could pass thru min_periods